docs/concepts/storage/volumes.md

```diff
@@ -553,9 +554,9 @@ __Important: Make sure you have an existing PortworxVolume with name `pxvol` bef
 More details and examples can be found [here](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/volumes/portworx/README.md)
 
 ### ScaleIO
-ScaleIO is a software-based storage platform that can use existing hardware to create clusters of scalable 
```

Comment:
Sorry about that. My text editor is just set up to trim whitespace at the end of lines. No change to the format of the rendered html, though.